device-connectors/src/testflinger_device_connectors/devices/iotscript/iot-auto-sanity/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
```

Comment:
Hi Paul
    This entire folder would be copied to zapper and installed then run on zapper (through zapper API), it would not be called by device-connector, this is an design and agreement in spec CR051 (https://docs.google.com/document/d/12cvc9l7v8ltQVlwx9hqqkoPxLdFW0nVEoDZ9vmPLMVw/edit?usp=sharing) that was created by Paolo.

Reply:
This seems a slightly bizarre design and not something I read from the spec. My understanding from the spec is that the device connector should be implementing rpyc calls to a remote process on the zapper. I don't see that reflected in this PR

Reply:
It's in the IoT method section, and due to this tool need to use usb serial to control target devices. Maciej think  run this tool in device connector through rpyc is not realistic. The method you method was our first approach but was rejected by Maciej and Paolo, so we have this version.

Reply:
@kiya956 @p-gentili @kissiel please make sure that the spec is in line with the implementation.